Makefile

```diff
@@ -9,7 +9,7 @@ VERSION=`git describe --tags --dirty`
 GO_VERSION_CHECK=`./scripts/check-go-version.sh`
 
 # The build target should stay at the top since we want it to be the default target.
-build: check-go-version pkg/web/ui/dist
+build: check-go-version pkg/web/ui/dist proto-generate
```

Comment:
@lovromazgon @maha-hajja wdyt about adding this to the `test` goal too, to make sure the code is always fresh?

Reply:
Makes sense. :+1:

Reply:
Seems like you forgot about this one.

Reply:
I didn't actually, I thought your answer was about my question adding it to the `test` goal and not the `build` goal. I'm viewing it from another angle: the fact that we change it infrequently means that it's easy to forget, and a few seconds are worth preventing outdated code. I see your point though, and don't feel strongly about it.

Reply:
Oh now I see this is on `build` 🤦 Well I still think we shouldn't run this if the generated code is already checked in, in most cases it won't do anything and this seems something that we should catch in PRs. Essentially the same reason why we don't run `go generate` before every build.

Reply:
I understand that point of view. :+1: I've removed the proto-generate before build just now.